### PR TITLE
[scan-osh] enable OCPBUGS workflow by default

### PR DESCRIPTION
**Pull request by ashwindasr**
group.yml will activate for all images. If we need to disable any particular image, we can override in that image config, by setting it to false

### CODE REVIEW

File: doozer/doozerlib/cli/scan_osh.py [hunk 1]
```diff
@@ -407,13 +407,14 @@ def get_packages_with_scan_issues(self, mapping: dict):
         return nvrs_with_scan_issues
 
     @staticmethod
-    def is_sast_jira_enabled(meta):
+    def is_sast_jira_disabled(meta):
         """
-        Check if the OCPBUGS workflow is enabled in ocp-build-data config
+        Check if the OCPBUGS workflow is enabled in ocp-build-data image meta config
         """
         flag = meta.config.get("external_scanners", {}).get("sast_scanning", {}).get("jira_integration", {}).get("enabled")
```
Comment (jupierce) [inline doozer/doozerlib/cli/scan_osh.py:414]: With "group as the default for image metadata", I would have expected:
```
flag = meta.config.external_scanners.sast_scanner.jira_integration.enabled
if flash is Missing:
  flag = meta.runtime. group_config.external_scanners.sast_scanning.jira_integration.enabled
```

and no code like honoring group globally: https://github.com/openshift-eng/art-tools/blob/0fa8d84a7fcad697e168bcbcb5ecbfbe7e0a0eba/doozer/doozerlib/cli/scan_osh.py#L662

Meaning that you could enable a singular image in a given release without turning it on for everything in group. Given the nature of the scanner, it is unlikely that we will require that configuration in practice, so I offer this for future reference only and won't hold up this PR. lgtm.

Comment (ashwindasr) [reply doozer/doozerlib/cli/scan_osh.py:414]: Noted, thanks for pointing this out!

I have to rewrite some of the code once osh-cli adds a feature that I requested. Will add this along with that change.

**Pull request merged**